dpkg-source: warning: extracting unsigned source package (/home/lucas/developers-reference_3.3.8-0.1.dsc)
diff -Nru developers-reference-3.3.8/debian/changelog developers-reference-3.3.8/debian/changelog
--- developers-reference-3.3.8/debian/changelog	2006-12-16 16:53:10.000000000 +0100
+++ developers-reference-3.3.8/debian/changelog	2008-02-06 00:00:45.000000000 +0100
@@ -1,3 +1,10 @@
+developers-reference (3.3.8-0.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Drop "Who can do an NMU" section. Closes: #XXX.
+
+ -- Lucas Nussbaum <lucas@lucas-nussbaum.net>  Wed, 06 Feb 2008 00:00:30 +0100
+
 developers-reference (3.3.8) unstable; urgency=low
 
   * mia-history is replaced by mia-query. Thanks, Christoph Berg.
diff -Nru /tmp/ILhXUxSWcw/developers-reference-3.3.8/developers-reference.sgml /tmp/Cvh2usLtFp/developers-reference-3.3.8/developers-reference.sgml
--- developers-reference-3.3.8/developers-reference.sgml	2006-11-12 12:05:22.000000000 +0100
+++ developers-reference-3.3.8/developers-reference.sgml	2008-02-06 00:00:24.000000000 +0100
@@ -3254,16 +3254,6 @@
 improperly orphaned package, please set the maintainer to ``Debian QA Group
 &lt;packages@qa.debian.org&gt;''.
 
-      <sect1 id="nmu-who">Who can do an NMU
-	<p>
-Only official, registered Debian Developers can do binary or source
-NMUs.  A Debian Developer is someone who has their key in the
-Debian key ring.  Non-developers, however, are encouraged to download
-the source package and start hacking on it to fix problems; however,
-rather than doing an NMU, they should just submit worthwhile patches
-to the Bug Tracking System.  Maintainers almost always appreciate
-quality patches and bug reports.
-
       <sect1 id="nmu-terms">Terminology
 	<p>
 There are two new terms used throughout this section: ``binary-only NMU''
